packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -127,6 +129,17 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 		return this._storageService.getSnapshotTree(version, scenarioName);
 	}
 
+	public async getSnapshot(
+		version?: IVersion,
+		snapshotFetchOptions?: ISnapshotFetchOptions,
+	): Promise<ISnapshot | undefined> {
+		assert(
+			this._storageService.getSnapshot !== undefined,
+			"getSnapshot api should exist in ContainerStorageAdapter",
+		);
+		return this._storageService.getSnapshot(version, snapshotFetchOptions);
```

Comment:
I see a lot of repetition like that.
Should this be done this way?

```ts
const getSnapshot = this._storageService.getSnapshot.bind(this._storageService);
```
OR using ?., if we want to preserve optionality.

Feels like too much code for trivial delegation

Reply:
Removed the asserts and throwing if api is not here. Also helps with having the correct return type as ISnapshot and not ISnapshot | undefined.